libsolidity/analysis/ReferencesResolver.cpp

```diff
@@ -111,17 +112,17 @@ void ReferencesResolver::endVisit(FunctionTypeName const& _typeName)
 	case VariableDeclaration::Visibility::External:
 		break;
 	default:
-		typeError(_typeName.location(), "Invalid visibility, can only be \"external\" or \"internal\".");
+		m_errorReporter.typeError(_typeName.location(), "Invalid visibility, can only be \"external\" or \"internal\".");
```

Comment:
@chriseth can you please review? Would be a good time to fix, otherwise we'll forget about it.